### PR TITLE
ARM32 bit fixes, for 64bit printf format specifier

### PR DESCRIPTION
**Pull request by antony-rheneus**
Signed-off-by: Antony Rheneus <arheneus@marvell.com>

**What I did**

**Why I did it**

**How I verified it**

**Details if related**

### CODE REVIEW

**Comment by antony-rheneus** [comment]
Need help in restarting the cs build, as it went into build failure as below.

```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 26: Unknown stage section "lock". Starting with version 0.5, steps in a stage must be in a ‘steps’ block. @ line 26, column 9.
           stage('Test') {
           ^

WorkflowScript: 26: Expected one of "steps", "stages", or "parallel" for stage "Test" @ line 26, column 9.
           stage('Test') {
           ^

```

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -1447,11 +1448,11 @@ bool PortsOrch::bake()
     }
 
     portCount = stoul(value);
-    SWSS_LOG_NOTICE("portCount = %lu, m_portCount = %u", portCount, m_portCount);
+    SWSS_LOG_NOTICE("portCount = %" PRIu64 ", m_portCount = %u", portCount, m_portCount);
     if (portCount != keys.size() - 2)
     {
         // Invalid port table
-        SWSS_LOG_ERROR("Invalid port table: portCount, expecting %lu, got %lu",
```
Comment (qiluo-msft) [inline orchagent/portsorch.cpp]: >%lu, got %lu" [](start = 65, length = 13)

The original format strings are correct. Please change back.

**Pull request merged**